meta-koi/recipes-android/android/android_koi-o.bb

```diff
@@ -0,0 +1,47 @@
+inherit gettext
+
+SUMMARY = "Downloads the Snapdragon Wear 2100/3100 /usr/libexec/hal-droid and /usr/include/android folders and installs them for libhybris"
+LICENSE = "CLOSED"
+
+SRC_URI = "https://dl.dropboxusercontent.com/s/4d8pkmd62rxl464/hybris-o-msm8909.tar.gz;name=hybris \
+    file://audio_policy.conf"
+SRC_URI[hybris.md5sum] = "039a387a3e4ccd09d28b05195280d162"
+SRC_URI[hybris.sha256sum] = "ddc725dd280d8c5f546ff47f727b67ea99def9a4883edeebfe0edfdc5a61a636"
+PV = "oreo"
+
+PACKAGE_ARCH = "${MACHINE_ARCH}"
+INHIBIT_PACKAGE_STRIP = "1"
+COMPATIBLE_MACHINE = "koi"
+INSANE_SKIP:${PN} = "already-stripped"
+S = "${WORKDIR}"
+B = "${S}"
+
+PROVIDES += "virtual/android-system-image"
+PROVIDES += "virtual/android-headers"
+
+do_install() {
+    # The stock audio policy contains invalid entries that cause the droid module to fail.
+    install -d ${D}${sysconfdir}/pulse
+    install -m 0644 ${WORKDIR}/audio_policy.conf ${D}${sysconfdir}/pulse/
+
+    install -d ${D}/usr/
+    cp -r usr/* ${D}/usr/
+
+    install -d ${D}${includedir}/android
+    cp -r include/* ${D}${includedir}/android/
+
+    install -d ${D}${libdir}/pkgconfig
+    install -m 0644 ${D}${includedir}/android/android-headers.pc ${D}${libdir}/pkgconfig
+    rm ${D}${includedir}/android/android-headers.pc
+    cd ${D}
+    ln -s system/vendor vendor
+}
+
+# FIXME: QA Issue: Architecture did not match (40 to 164) on /work/dory-oe-linux-gnueabi/android/lollipop-r0/packages-split/android-system/system/vendor/firmware/adsp.b00 [arch]
```

Comment:
I think you could fix this with one of the `INSANE_SKIP`, maybe if you add arch after already-stripped l15 ?

Reply:
Alright! Maybe we could add a comment clarifying that then ?

Reply:
I'm not adding this here, unless this has already been added to all other recipes. I think that should be one commit that adds the clarification to all affected ports.